content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,278 @@
+4. Use the actions available in the upper left corner to enrich your scenario:
+    {{< img src="synthetics/browser_tests/browser_test_step.png" alt="Browser Test steps" responsive="true" style="width:75%;">}}
+
+    **Note**: **Your last browser test step must be an Assertion**, this will ensure your test ended up on the expected page and found the expected element. This will allow your test to use a different set of keys to perform the recorded journey and confirm it landed on the expected element although some changes on the UI might have happened since the test was first recorded.
+
+5. Click the **Variables** button under *Add New*. Select `Pattern`, `Element`, `Global Variable`, or `Email` from the dropdown menu. For example, select `Email` and Datadog will create a special email variable that allows you to test that an email was sent by the browser test and received by one of our dedicated email servers.
+
+    **Note**: Email assertions require the use of an email variable. For example, if you want to check that a body contains a specific name, create an email variable as noted above and add it to your assertion.
+
+6. Use the email variable as an input step in a test to trigger an email.
+
+    {{< img src="synthetics/testing-variable.png" alt="Browser Test steps" responsive="true" style="width:100%;">}}
+
+7. Click the **Assertion** button under *Add New*. Select an assertion, such as `Test that an email was received`.
+    
+    [Assertions][13] allow you to check whether an element, some content, or some text is available on the current page. With assertions, you can check the URL of a page or verify that a specific email was sent. For example, you can create an assertion that checks whether an email was sent and whether specific values (string, number, regex) are present within the email subject or body. You first need to create an email variable to be able to use the email assertion. 
+    
+8. Select the subject condition and click **Use variable**. Select the email variable you created. This assigns the email variable you created to the assertion. Select the body condition and click **Use variable**. Select the email variable you created.
+9. Once you have finished your Scenario, click on **Save and Launch Test**.
+
+    {{< img src="synthetics/browser_tests/context-panel.gif" alt="Context panel" responsive="true" style="width:100%;">}}
+
+A **browser test** homepage will automatically populate after save. This page includes property information, historical graphs for response time and uptime, sample results, and all events and test results. Sample results include errors, resources, and traces. To review this information, click on the `error`, `resources`, or `traces` button located next to the sample result action.
+
+### Set up a private location
```

Comment:
```suggestion
## Set up a private location
```

Reply:
Setting up a private location should not be part of the "configure a test" section, it's something different, it's about creating a whole location (that will then be used to run tests)